cocktail-dkg.md

```diff
@@ -745,8 +746,8 @@ function Round1(i, t, n, cs, context, d_i, AllPublicKeys):
                   of H6 as an Input Keying Material to two more hash function calls, which will be used
                   to derive a nonce and key. (Here, H() refers to, e.g., sha256):
                   - ikm = H6(ecdh_secret, E_i, P_j, context)
-                  - key = H("COCKTAIL-derive-key" || ikm)
-                  - nonce = H("COCKTAIL-derive-nonce" || ikm)[0:24]
+                  - key = H("COCKTAIL-derive-key" \parallel ikm)
+                  - nonce = H("COCKTAIL-derive-nonce" \parallel ikm)[0:24]
```

Comment:
```suggestion
                  - $key = H("COCKTAIL-derive-key" \parallel ikm)$
                  - $nonce = H("COCKTAIL-derive-nonce" \parallel ikm)[0:24]$
```

Reply:
And in fact most of the other elements of this list - they all should be between `$`...`$`.